src/view/view-layout.js

```diff
@@ -0,0 +1,75 @@
+import _ from 'underscore';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import Sink from './sink';
+import sinkLayoutGen from './sink-layout-gen';
+
+class View extends Component {
```

Comment:
Not quite grasping the role of this component. Does it need to be its own thing or can it be folded into ViewLayout?